src/PresenterLayer/KeyMotorPresenter/KeyMotorPresenter.h

```diff
@@ -17,9 +17,14 @@ class KeyMotorPresenter : public QObject
 signals:
     void motorZeroReceived(KeyMotor);
     void motorOneReceived(KeyMotor);
+    void motorZeroBusPowerReceived(double);
+    void motorOneBusPowerReceived(double);
+    void motorActualSpeedReceived(double);
```

Comment:
Re-add this between `emit motorSetCurrentReceived` and `emit motorBusVoltageReceived`, so it doesn't show up in the diff